starters/oasp4j-batch-starter/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.oasp.java.dev</groupId>
+    <artifactId>oasp4j-starters</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
+  <groupId>io.oasp.java.starters</groupId>
```

Comment:
great that you added a separate structure/folder/parent POM for the starters. But do we want to have a separate groupId for that? Maybe this is causing more problems as users then have to pick the corrent groupId while otherwise the groupId would always be the same for all modules what is IMHO more simple and less error-prune.